utilities/cli/util.py

```diff
@@ -1419,6 +1382,68 @@ def setup_ngc_cli(dry_run: bool = False) -> None:
         fatal(f"Failed to install NGC CLI: {e}")
 
 
+def setup_sccache(min_version: str = "0.12.0", dry_run: bool = False) -> None:
```

Comment:
this could be part of the setup scripts? https://github.com/nvidia-holoscan/holohub/tree/main/utilities/setup cc @tbirdso